.github/workflows/linkchecker.yml

```diff
@@ -0,0 +1,11 @@
+name: Check links
```

Comment:
Don't we already have a link-checker?

Reply:
ohhh that's the SDK right, if so, ignore.

Reply:
not in the ci, this is the same github action that is in the cosmos-sdk